### ISSUE TITLE
Versions page not found

### ISSUE THREAD

**Issue by Sulyci** [angular/angular#54300]
### Describe the problem that you experienced

The [See Angular's version compatibility](https://angular.dev/versions) page is not found.

### Enter the URL of the topic with the problem

https://angular.dev/tools/cli/setup-local#dependencies

### Describe what you were looking for in the documentation

I was looking for the version compatibility page.

### Describe the actions that led you to experience the problem

In [Local Setup](https://angular.dev/tools/cli/setup-local) page there is a hyperlink to [See Angular's version compatibility](https://angular.dev/versions) page. I clicked on this link but the page is not found. 

### Describe what you want to experience that would fix the problem

It would be refreshing if I could see the version compatibility page.

### Add a screenshot if that helps illustrate the problem

<img width="852" alt="Képernyőfotó 2024-02-06 - 23 28 32" src="https://github.com/angular/angular/assets/6008600/1fdee43e-574d-4b23-902d-c4c167db9da4">

### Provide any additional information here in as much as detail as you can

```true
I am using a MacBook Pro 14" M3 modell (16GB RAM), and I am using the newest Safari.
```

**Comment by JeanMeche** [comment]
Hi, thanks for reporting this issue !
It has been fixed by #52585 and is pending deployement on our side !

**Comment by Sulyci** [comment]
Hi, That's great! Thank you!
